memecity.game/Game/Generate/NPC/NPCGenerator.cpp

```diff
@@ -1,76 +1,236 @@
 #include "NPCGenerator.h"
-#include "..\..\Components.h"
+#include "../../Components.h"
 #include <stdlib.h>     /* srand, rand */
 #include <time.h>       /* time */
-#include "..\..\..\Assets.h"
+#include "../../../Assets.h"
 
 using namespace memecity::engine;
 using namespace memecity::engine::ecs;
 
 namespace generate {
 	int NPCGenerator::random_int(int max) {
-		srand(time(NULL));
-		if (max <= 0) {
-			return 0;
+		if (max <= 0 || 5 > max) {
+			return 5;
 		}
-		return (rand() % max);
+		return (5 + (rand() % (max)));
 	}
 
-	const memecity::engine::ecs::Entity& NPCGenerator::generate(int maxlevel, float x, float y) {
-		this->level = rand() % maxlevel + 1;
+
+	const memecity::engine::ecs::Entity& NPCGenerator::generate_police_npc(float x, float y) {
+		this->name = "Handhaving";
+		this->level = rand() % 1;
+		//TODO:: change on map
+		this->range_of_fighting = rand() % 30 + 60;
```

Comment:
ik zou de minimum waarde in een variable zette zodat het duidelijk is wat het inhoud